vcd/resource_vcd_nsxt_network_dhcp_test.go

```diff
@@ -130,5 +138,7 @@ resource "vcd_nsxt_network_dhcp" "pools" {
     start_address = "7.1.1.130"
     end_address   = "7.1.1.140"
   }
+
+  dns_servers = ["8.8.4.4", "8.8.8.8"]
```

Comment:
So if I understood correctly, this means that order matters for this attribute, right?

Reply:
That is right!
Those are the DNS servers that are being server through DHCP. Normally you will use the first one and have the second one as backup.

Also in the VCD UI you can see the order if you change it.

Reply:
Great, thanks!